src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -585,6 +585,10 @@ private void EndParallelParameterSet()
             _taskCollection.Complete();
```

Comment:
@PaulHigin, your last commit had 1 failures in `PowerShell-CI-linux`
ForEach-Object -Parallel Functional Tests.Verifies job queuing and throttle limit
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1: line 337
337:         $job | Remove-Job -Force

```